edge-hub/test/Microsoft.Azure.Devices.Routing.Core.Test/routeFactory/RouteFactoryTest.cs

```diff
@@ -64,6 +64,10 @@ public void TestParseRouteWithSystemEndpoint(string routeString, IMessageSource
         [InlineData(@"FORM /messages/modules/adapter INTO brokeredEndpoint(""/modules/alertLogic/inputs/in"")")]
         [InlineData(@"SELECT FROM /messages/modules/adapter INTO brokeredEndpoint(""/modules/alertLogic/inputs/in"")")]
         [InlineData(@"FROM INTO brokeredEndpoint(""/modules/alertLogic/inputs/in"")")]
+        [InlineData(@"FORM /* INTO brokeredEndpoint(""/modules/alertLogic/inputs/in')")]
```

Comment:
`FORM` -> `FROM` (the intention is for the parse to fail because of the mismatched quotes, not the misspelling), and `/messages` -> `/*` for consistency.

Reply:
Done